templates/students/attendance.html

```diff
@@ -6,145 +6,198 @@
-                                </div>
-                            </td>
-                        </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-        </div>
-        <!-- <p class="text-center text-danger mt-1">{{cwhw_error}}</p> -->
-        <div class="row justify-content-center mb-4">
-            <div class="col-md-4">
-                <button name="submit" value="stu-att" class="btn btn-primary btn-lg btn-block" type="submit">Submit</button>
-            </div>
-        </div>
-    </form>
-    {% endif %}
-</div>
+                <!-- <p class="text-center text-danger mt-1">{{cwhw_error}}</p> -->
+                <div class="row justify-content-center mb-4">
+                    <div class="col-md-4">
+                        <button name="submit" value="stu-att" class="btn btn-primary btn-lg btn-block" type="submit">
+                            Submit
+                        </button>
+                    </div>
+                </div>
+            </form>
+        {% endif %}
+    </div>
 
 {% endblock %}
 
 {% block scripts-sidebar %}
 
-<!-- AJAX for Student Attendance -->
-
-<script>
-    // $("#stu-att-class option:selected").removeAttr('selected');
-    $("#stu-att-class option:first").attr('selected', 'selected');
-    $("#stu-att-class").change(function () {
-        let stu_class = $(this).val();
-
-        $.ajax({
-            url: "{% url 'students:ajax_attendance' %}",
-            data: {
-                'stu_class': stu_class,
-            },
-            dataType: 'json',
-            success: function (data) {
-                let stu_att_table = '<thead><tr><th>Student Name</th><th>Class</th><th>Mark Present</th></tr></thead><tbody>';
-                let checked = '';
-
-                Object.entries(data).forEach(([key, [stu_id, name, school_class, present]]) => {
-                    if (present) checked = 'checked';
-                    else checked = '';
-
-                    if(school_class == 13) school_class = "JNV_4"
-                    else if(school_class == 14) school_class = "JNV_5"
-
-                    stu_att_table += '<tr><td>' + name + '</td><td>' + school_class + '</td> \
+    <script>
+        function attendance(student_id, checkboxID) {
+            let student = document.getElementById('s' + checkboxID);
+            let is_present = student.checked;
+            console.log(is_present);
+            $.ajax({
+                url: "{% url 'students:ajax_mark_attendance' %}",
+                data: {
+                    'std_id': student_id,
+                    'is_present': is_present,
+                },
+                dataType: 'json',
+                success: function (data, status) {
+                    $('#alert-toast-div > div').html(
+                        `<div class="toast" id="alert_toast" role="alert" aria-live="assertive" aria-atomic="true" data-delay="3500">
```

Comment:
Change the `data-delay` to 500. As this toast is going to appear again and again, we don't need to show it for 3.5 sec.